modules/server/src/graphqlRoutes.js

```diff
@@ -259,7 +259,7 @@ export const createSchemasFromConfigs = async ({
 			types: typesWithMappings,
 		});
 
-		if (false) {
+		if (true) {
```

Comment:
at this point, might well remove the "if", no?

Reply:
I'm going to leave this in. With the upcoming "mode" configuration , this will still be an `if`
We won't add any network aggregation functionality if config is just normal mode

Reply:
Probably want to drop a `// TODO` here then just to track the expected change. In this very obvious case I don't think we'll lose track that a change is needed, but its useful to let people reading/helping with reviews know what the plan is for this.